### ISSUE TITLE
SOR: support ETH address (0xeee...ee)

### ISSUE THREAD

**Issue by gmbronco** [beethovenxfi/beethovenx-backend#587]
```
{
  sorGetSwaps(
    swapType: EXACT_IN
    tokenIn: "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
    tokenOut: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
    swapAmount: "1"
    chain: MAINNET
    swapOptions: { maxPools: 0 }
  ) {
    returnAmount
  }
}
```

throws: Missing token from tokenService 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

**Comment by gmbronco** [comment]
daniel: should never do that path through the SOR
the frontend should just change to wrap/unwrap

**Comment by gmbronco** [comment]
We should be using NATIVE_ASSETS as input / output tokens

https://github.com/balancer/b-sdk/blob/main/src/utils/constants.ts/#L167

**Comment by gmbronco** [comment]
fixed by #597
